device.mk

```diff
@@ -461,6 +461,12 @@ PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.radio.data_no_toggle=1
 
+# Enable AAudio MMAP/NOIRQ data path.
+# 2 is AAUDIO_POLICY_AUTO so it will try MMAP then fallback to Legacy path.
```

Comment:
By using this method, if you choose not to add AAudio to your own kernel this will allow it to work on mine, while falling back to the "normal" paths for your users who do not use Zest.